Artifacts.toml

```diff
@@ -44,3 +44,10 @@ git-tree-sha1 = "043f9354b961fd3ef6ac4cf71d0c99930e177a92"
     [[bonan_richards_eqn.download]]
     sha256 = "50f1739dfd8193742488f249bd1ba8a157dfc7c1d8a27392ba7edbe9b4c0db03"
     url = "https://caltech.box.com/shared/static/4jcz9c1lp6rt750q28kx3qvihi8393tv.gz"
+
+[twostr_test]
+git-tree-sha1 = "b0986aa28e7970a9c080461f5e52fb125cda9c3d"
+
+    [[twostr_test.download]]
+    sha256 = "c972f3ff4e041a4443c6e970021f1084a1cc176c2aeeda453f3351ef7cffa60b"
+    url = "https://caltech.box.com/shared/static/zh7wb0ar8kwg4sio64id9e6otzcsfhad.gz"
```

Comment:
I still have to look at the PR in ClimaArtifacts, but if this artifact is only used for tests, we should make it a test artifact.
Namely, add an Artifacts.toml in the test folder and add this snippet in runtests
```
import Pkg, Artifacts
# Download test-only artifacts
#
# (Currently not natively supported by Julia)
artifacts_toml = joinpath(@__DIR__, "Artifacts.toml")
artifacts = Artifacts.select_downloadable_artifacts(artifacts_toml)
for name in keys(artifacts)
    Pkg.Artifacts.ensure_artifact_installed(name, artifacts[name], artifacts_toml)
end
```